src/alfasim_sdk/_internal/alfacase/case_to_alfacase.py

```diff
@@ -40,6 +40,8 @@ def format_list(values: List[Any], *, enable_flow_style: bool = False):
     """
     import ruamel
 
+    # TODO: arthur: Review this workaround before merging!!!
```

Comment:
TODO: Review this before merging

Reply:
I would recommend to check on how to enable flow style for a section of the document.

Reply:
Yeah thanks... Currently I have no idea how to handle this. It's not working, tests in ALFAsim are passing. But `alfasim-sdk` is now broken.